all-resources.yaml

```diff
@@ -0,0 +1,200 @@
+# Before running this file, create required secrets. See README.md for details
```

Comment:
I would probably assume that the user has already ran `dispatch create seed-images`.  If not we should point folks to that to create the generic images, and add on top of them the stuff for the demo.

Reply:
sounds good, I added instructions to use seed-images and changed the resource file.